src/main/java/uk/gov/hmcts/sptribs/caseworker/util/DocumentListUtil.java

```diff
@@ -121,12 +127,16 @@ private static List<CaseworkerCICDocument> getCloseCaseDocuments(CaseData caseDa
 
     private static List<CaseworkerCICDocument> getHearingSummaryDocuments(CaseData caseData) {
         List<CaseworkerCICDocument> hearingSummaryDocs = new ArrayList<>();
-        if (null != caseData.getListing() && null != caseData.getListing().getSummary()
-            && !CollectionUtils.isEmpty(caseData.getListing().getSummary().getRecFile())) {
-            for (ListValue<CaseworkerCICDocument> document : caseData.getListing().getSummary().getRecFile()) {
-                hearingSummaryDocs.add(document.getValue());
-            }
-        }
+
+        Stream.ofNullable(caseData.getHearingList())
+            .flatMap(Collection::stream)
+            .map(ListValue::getValue)
+            .filter(hearing -> !isNull(hearing.getSummary()) && isNotEmpty(hearing.getSummary().getRecFile()))
```

Comment:
In a list, if any one of the element is null or empty, isNotEmpty() will return false as it would expect all the elements to have non-empty values.
If we want to skip the null and empty values in the list and  want to process non-empty elements in the list, we shouldn't be using CollectionUtils methods. Please validate.

Reply:
No it checks if the list is null or empty:

```
public static boolean isNotEmpty(final Collection<?> coll) {
        return !isEmpty(coll);
    }

public static boolean isEmpty(final Collection<?> coll) {
        return coll == null || coll.isEmpty();
    }
```

Reply:
I can change to use a different library if you're not keen on CollectionUtils

Reply:
can you create a unit test case by adding null, empty and non-empty values to a list and observe the behaviour?